static/js/server_events_dispatch.js

```diff
@@ -496,10 +498,12 @@ export function dispatch_normal_event(event) {
                             settings_org.sync_realm_settings("signup_notifications_stream_id");
                         }
                     }
+                    stream_list.update_subscribe_to_more_streams_link();
                     break;
                 default:
                     blueslip.error("Unexpected event type stream/" + event.op);
-                    break;
+                    // We return here to avoid the extra processing.
+                    return;
```

Comment:
Let's remove this hunk; it's not wrong/confusing.

Reply:
Updated the PR, thanks!